scripts/eloot.lic

```diff
@@ -2610,20 +2609,24 @@ module ELoot
 
         start_silvers = ELoot.silver_check
         while box.contents.any? { |obj| obj.name =~ /silver coins/ }
-          res = ELoot.get_res("get coins from ##{box.id}", /You gather the remaining|Get what|You can only collect/)
-          if res =~ /You gather the remaining/
-            data[location] += ELoot.silver_check - start_silvers
-            ELoot.wait_rt
-            break
-          elsif res =~ /You can only collect/
-            data[location] += ELoot.silver_check - start_silvers
-            ELoot.wait_rt
-            room = Room.current.id
-            ELoot.silver_deposit
-            ELoot.go2(room)
+          unless ELoot.data.charm.nil?
+            res = ELoot.get_res("point ##{ELoot.data.charm.id} at ##{box.id}", /You summon/)
```

Comment:
Functionally the charm works great, but due to splitting it out and not checking the output for the amount of silvers grabbed, it's not updating the silver breakdown chart at the end. Other than that, everything working great!

Reply:
fixed